optimum/onnx/graph_transformations.py

```diff
@@ -14,25 +14,32 @@
 import copy
 import os
 from pathlib import Path
-from typing import Optional, Union
+from typing import TYPE_CHECKING, List, Optional, Union
 
 import onnx
 from onnx import ModelProto
 
 from ..utils import logging
 from .transformations_utils import (
     _create_name_sharing_dict,
+    _deduplicate_gather_matmul,
     _deduplicated_cross_model_initializers,
     _find_duplicate_initializers,
+    _find_matching_initializers,
     _get_all_inputs,
     _get_onnx_opset,
+    _get_weights_to_tie,
     _remove_redundant_initializers,
     _replace_input_names,
     _unify_onnx_outputs,
     cast_int64_tensorproto_to_int32,
 )
 
 
+if TYPE_CHECKING:
+    import torch.nn as nn
```

Comment:
Should this also be under a `is_torch_available()`, or can we assume it's available here?

Reply:
We can probably assume that a dev has all the dependencies installed so probably not needed.